src/app/wallets/index.ts

```diff
@@ -23,3 +23,9 @@ export const listWalletsByAccountId = async (
 ): Promise<Wallet[] | RepositoryError> => {
   return WalletsRepository().listByAccountId(accountId)
 }
+
+export const getAllWalletIds = async (
+  walletCurrency: WalletCurrency,
+): Promise<WalletId[] | RepositoryError> => {
+  return WalletsRepository().listAllWalletIds(walletCurrency)
+}
```

Comment:
could even be reduced by the following:

```suggestion
): Promise<WalletId[] | RepositoryError> => 
 WalletsRepository().listAllWalletIds(walletCurrency)
```